### DISCUSSION TITLE
How to change timezone on SnipeIT

### DISCUSSION THREAD

**Discussion by NadjaUniver** [snipe/snipe-it#13514] [Q&A] [answered]
My timezone is going 2h backwards, how to change it?

**Comment by tchangth** [comment, marked as answer]
i usually to sudo timedatectl set-timezone 'Europe/London'

**Reply by NadjaUniver** [reply]
Thanks, it works for me :)

**Reply by AsyrafAzizan** [reply]
can i use this command in my command prompt?
